minio/api.py

```diff
@@ -2601,9 +2603,8 @@ def delete_object_lock_config(self, bucket_name):
         Example::
             client.delete_object_lock_config("my-bucket")
         """
-        self.set_object_lock_config(
-            self, bucket_name, ObjectLockConfig(None, None, None),
-        )
+        lock_config = ObjectLockConfig(None, None, None)
+        self.set_object_lock_config(bucket_name, lock_config)
```

Comment:
As you are unable to keep it one liner, just remove extra `self`.

Reply:
nope, the linter complains about the line length (over 80 chars) in such case

Reply:
I am saying remove `self` usage in original code. There is no need to have `lock_config` variable.

Reply:
I did and got these linter errors: https://github.com/minio/minio-py/runs/1714888958

Shall I change linter settings and increase the maximum length of a single line?

Reply:
I am saying below
```py
        self.set_object_lock_config(
            bucket_name, ObjectLockConfig(None, None, None),
        )
```

Reply:
ok, reverted it back